sdk/src/invariant.ts

```diff
@@ -811,13 +817,24 @@ export class Invariant {
     )
   }
 
-  async getLiquidityTicksAmount(poolKey: PoolKey): Promise<bigint> {
-    return sendQuery(
+  // Query needs to be split in the case where tickSpacing = 1, otherwise a single query will fit within the gas limit
+  async getLiquidityTicksAmount(
```

Comment:
Just a heads up: this function will only be used in an alternative scenario on the frontend (Of course, we'll keep this function)